frontend/src/views/TitleDetail.vue

```diff
@@ -59,6 +59,17 @@
               >
             </td>
           </tr>
+          <tr>
+            <td>Books</td>
+            <td
```

Comment:
this won't work as soon as you have more than one book for the title: this table is 2-cols large so it will break the rendering.
You can use a single cell and if you have more than a single book, use a list `ul` inside the cell